app/Models/Entry.php

```diff
@@ -480,40 +480,37 @@ public function isDay(int $day, int $today): bool {
 	public static function getContentByParsing(string $url, string $path, array $attributes = [], int $maxRedirs = 3): string {
 		$html = getHtml($url, $attributes);
 		if (strlen($html) > 0) {
-			require_once(LIB_PATH . '/lib_phpQuery.php');
-			/**
-			 * @var phpQueryObject @doc
-			 */
-			$doc = phpQuery::newDocumentHTML($html);
+			$doc = new DOMDocument();
+			$doc->loadHTML($html, LIBXML_NONET | LIBXML_NOERROR | LIBXML_NOWARNING);
+			$xpath = new DOMXPath($doc);
 
 			if ($maxRedirs > 0) {
 				//Follow any HTML redirection
-				/**
-				 * @var phpQueryObject @metas
-				 */
-				$metas = $doc->find('meta[http-equiv][content]');
+				$metas = $xpath->query('//meta[@content]');
```

Comment:
Does this deal with random whitespace that doesn't affect HTML?

Reply:
Yes